public/app/core/components/RolePicker/RolePicker.tsx

```diff
@@ -140,7 +145,14 @@ export const RolePicker = ({
   }
 
   return (
-    <div data-testid="role-picker" style={{ position: 'relative', width: ROLE_PICKER_WIDTH }} ref={ref}>
+    <div
+      data-testid="role-picker"
+      style={{
+        position: 'relative',
+        maxWidth: maxWidth,
```

Comment:
```suggestion
        maxWidth,
```
This can be simplified.

Reply:
Thanks!